src/modules/analysis/AnalysisViewComponent.js

```diff
@@ -58,6 +54,18 @@ const getTableRow = a => {
   );
 };
 
+const FieldGroup = ({ id, label, md, value }) => (
```

Comment:
In the example given by you, FieldGroup is not standard component it is custom component and they given a random name.